src/main/java/edu/cornell/kfs/pmw/batch/service/impl/PaymentWorksWebServiceCallsServiceImpl.java

```diff
@@ -397,14 +402,31 @@ private Invocation buildMultiPartRequestForSupplierUpload(Client client, URI uri
         multiPart.setMediaType(MediaType.MULTIPART_FORM_DATA_TYPE);
         multiPart.bodyPart(csvPart);
         
-        return client.target(uri)
-                .request()
+        WebTarget target = client.target(uri);
+        Invocation.Builder requestBuilder = target.request();
+        disableRequestChunkingIfNecessary(client, requestBuilder);
+        
+        return requestBuilder
                 .accept(MediaType.APPLICATION_JSON_TYPE)
                 .header(PaymentWorksWebServiceConstants.AUTHORIZATION_HEADER_KEY, 
                              PaymentWorksWebServiceConstants.AUTHORIZATION_TOKEN_VALUE_STARTER + getPaymentWorksAuthorizationToken())
                 .buildPost(Entity.entity(multiPart, MediaType.MULTIPART_FORM_DATA_TYPE));
     }
 
+    private void disableRequestChunkingIfNecessary(Client client, Invocation.Builder requestBuilder) {
+        if (client instanceof org.apache.cxf.jaxrs.client.spec.ClientImpl) {
+            if (LOG.isDebugEnabled()) {
+                LOG.debug("disableRequestChunkingIfNecessary: Explicitly disabling chunking because KFS is using a CXF JAX-RS client");
+            }
+            ClientConfiguration cxfConfig = WebClient.getConfig(requestBuilder);
+            HTTPConduit conduit = cxfConfig.getHttpConduit();
+            HTTPClientPolicy clientPolicy = conduit.getClient();
+            clientPolicy.setAllowChunking(false);
+        } else if (LOG.isDebugEnabled()) {
+            LOG.debug("disableRequestChunkingIfNecessary: The JAX-RS client implementation does not need to explicitly disable chunking");
```

Comment:
A couple thoughts here.

As coded now, we will only not enable chunking when the client is something other than org.apache.cxf.jaxrs.client.spec.ClientImpl .  If we stay with that, I think want to include the instance type of the client object.  Also, should it be something other than debug, something would have changed that we should probably be made aware of I think

If we want to add further restrictions, we should add the vendor count to the log statement, and surround it with if (LOG.isDebugEnabled())

Reply:
Ok, I'll work on updating this to add more log information about the JAX-RS client in use, and to change the log level accordingly.

Also, the calling code in PaymentWorksUploadSuppliersServiceImpl is already logging the number of vendors to upload, so we don't need to log it again here. Doing so here instead would be possible, but would require changing various method signatures to accept new or different arguments.